api/src/main/java/javax/enterprise/inject/spi/Annotated.java

```diff
@@ -94,6 +113,8 @@
     /**
      * <p>
      * Determine if the program element has an annotation of a certain annotation type.
+     *
+     * If the <tt>annotationType</tt> is repeatable, then returns true if at least one instance is available
```

Comment:
Usually we follow JLS and JDK rule but there are exception. My POV here is there's more added value in @johnament approach than following how the JDK.